modules/doc/content/help/faq/conda_issues.md

```diff
@@ -104,14 +104,20 @@ Conda issues can be the root cause for just about any issue on this page. Scroll
 
   +Hint:+ For Linux, this will most likely be `conda init bash`. For Macintosh, this can either be bash or zsh.
 
+  !alert note title=mamba init
+  When using `mambaforge3` you might also need to perform a `mamba init` in order to properly use the
+  `mamba` command to install packages. We recommend this as an alternative to conda, as mamba can take
+  advantage of multiple cores to perform tasks. This means it will generally be faster at performing
+  installations than standard conda. Otherwise, they are interchangeable.
```

Comment:
This means *mamba* will generally be faster......Otherwise, mamba and conda are interchangeable?

Reply:
oh, I'm trying to suggest inline word changes but have been able to suggest a git change. Mostly I am not a fan of pronoun references ;-)

Reply:
Ah! I got you - I am making some more changes to the instructions based on @milljm's in-person feedback, so I will make this change too.